ofonotest/qml/ofonotest/main.qml

```diff
@@ -1,74 +1,148 @@
 import QtQuick 2.0
 import QOfono 0.2
 
-
 Rectangle {
     width: 480
     height: 560
-    Text {
-        id: textLine
-        anchors.centerIn: parent
-    }
-    Text {
-        id: textLine2
-        anchors.top: textLine.bottom
-        text: manager.available ? netreg.name : "Ofono not available"
+
+    Column {
+
+        anchors.fill: parent
+
+        Text {
+            text: context1.active ? "online" : "offline"
+        }
+
+//        Text {
+//            text:  "modem Manufacturer " + modem1.manufacturer
+//        }
+
+        Text {
+            id: textLine2
+
+            text: manager.available ? netreg.name : "Ofono not available"
+        }
+
+        Text {
+            text: "simManager.pinRequired " + simManager.pinRequired
+            visible: simManager.pinRequired !== OfonoSimManager.NoPin
+        }
+
+        Repeater {
+            model: manager.modems
+
+            Text {
+                text: modelData
+                font.bold: modelData === manager.defaultModem
+            }
+
+        }
+
     }
+
     MouseArea {
         anchors.fill: parent
         onClicked: {
-          context1.active = !context1.active
+            context1.active = !context1.active;
+            console.log("context1.active " + context1.active);
         }
     }
 
     OfonoManager {
         id: manager
+
         onAvailableChanged: {
-            console.log("Ofono is " + available)
-           textLine2.text = manager.available ? netreg.currentOperator["Name"].toString() :"Ofono not available"
+            console.log("Ofono is " + available);
+            console.log("netreg.currentOperatorPath: " + netreg.currentOperatorPath);
+            textLine2.text = (manager.available && netreg.currentOperator !== undefined) ? netreg.currentOperator["Name"].toString() : "Ofono not available";
```

Comment:
Hm, maybe not regression but trying this I'm getting console output of 
[D] onAvailableChanged:55 - Ofono is true

and this saying "Ofono not available" on the UI. Supposedly because of:
[D] onAvailableChanged:56 - netreg.currentOperatorPath:

Reply:
Actually, I am not sure what was original purpose. I can guess that it should write mobile operator name, but the value is accessible via `netreg.name`. The array seems to be not populated. I didn't check how this interface changed in the past.

Reply:
Personally, I would avoid assignment `textLine2.text = `, because it breaks binding defined above (I have changed the behaviour in recent force push in this merge request). As result it is showing network operator now (KAKTUS in my case, but can be T-Mobile or AT&T as well)